includes/backup_migrate.bee.inc

```diff
@@ -0,0 +1,388 @@
+<?php
+
+
+/**
+ * @file
+ * Bee commands for backup and migrate.
+ */
+
+/**
+ * Implementation of hook_bee_command().
+ */
+function backup_migrate_bee_command() {
+  $items['bam-backup'] = array(
+    'description' => bt('Backup the site\'s database with Backup and Migrate.'),
+    'callback' => 'backup_migrate_backup_bee_callback',
+    'group' => 'backup_migrate',
+    'arguments' => array(
+      'source'        => "Optional. The id of the source (usually a database) to backup. Use 'bee bam-sources' to get a list of sources. Defaults to 'db'",
+      'destination'   => "Optional. The id of destination to send the backup file to. Use 'bee bam-destinations' to get a list of destinations. Defaults to 'manual'",
+      'profile'       => "Optional. The id of a settings profile to use. Use 'bee bam-profiles' to get a list of available profiles. Defaults to 'default'",
+    ),
+    'optional_arguments' => array(
+      'source',
+      'destination',
+      'profile',
+    ),
+    'aliases' => array('bb'),
+    'bootstrap' => BEE_BOOTSTRAP_FULL,
+    'examples' => array(
+      'bee bam-backup' => 'Backup the default databse to the manual backup directory using the default settings.',
+      'bee bam-backup db scheduled mysettings' => 'Backup the database to the scheduled directory using a settings profile called "mysettings"',
+      'bee bam-backup files' => 'Backup the files directory to the manual directory using the default settings. The Backup and Migrate Files module is required for files backups.',
+    ),
+  );
+
+  $items['bam-restore'] = array(
+    'description' => bt('Restore the site\'s database with Backup and Migrate.'),
+    'callback' => 'backup_migrate_restore_bee_callback',
+    'group' => 'backup_migrate',
+    'arguments' => array(
+      'source'        => "Required. The id of the source (usually a database) to restore the backup to. Use 'bee bam-sources' to get a list of sources. Defaults to 'db'",
+      'destination'   => "Required. The id of destination to send the backup file to. Use 'bee bam-destinations' to get a list of destinations. Defaults to 'manual'",
+      'backup'        => "Required. The id of a backup file restore. Use 'bee bam-backups' to get a list of available backup files.",
```

Comment:
There is no 'bee bam-backups' command. Did you mean 'bam-created'?

Reply:
Yes that is my mistake. You are correct, it is `bee bam-created`. I will update the branch and push another update.